bibliopixel/drivers/serial/devices.py

```diff
@@ -81,6 +105,11 @@ def set_device_id(self, dev, id, baudrate=921600):
             raise_error(ord(resp))
 
     def get_device_id(self, dev, baudrate=921600):
+        """Get device ID at given address/path.
```

Comment:
"Address/path"?  Let's just stick with "address", and we can later put an entry in our glossary for "serial address"